tests/unit/test_commands.py

```diff
@@ -553,9 +553,9 @@ def test_parse_args_plan(args, expected_CLIargs):
                 auto_approve=False,
                 backup=True,
                 force=False,
-                raw_machines=None,
-                raw_hostnames=["1", "2,3"],
-                raw_availability_zones=None,
+                machines=None,
+                hostnames={"1", "2", "3"},
```

Comment:
non-blocking: since you are using set, it would be good to add a test for an input like `-m 1,2,3 -m 3,5`